apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.ts

```diff
@@ -53,6 +53,21 @@ router.get(
       assessmentQuestionId: res.locals.assessment_question.id,
     });
 
+    const instance_questions = await queryRows(
+      sql.select_instance_questions_manual_grading,
+      {
+        // TODO: improve typing of res.locals
+        assessment_id: (res.locals as any).assessment.id,
+        assessment_question_id: res.locals.assessment_question.id,
```

Comment:
You already have `typedAsyncHandler<'instructor-assessment-question'>`, why not use it here?